applications/crossbar/src/modules/cb_braintree.erl

```diff
@@ -223,6 +229,18 @@ validate_credits(Context, ?HTTP_PUT) ->
                               maybe_charge_billing_id(Amount, Context1)
                       end.
 
+validate_token(Context, ?HTTP_GET) ->
+    try braintree_client_token:get_client_token(cb_context:account_id(Context)) of
+        Token ->
+            JObj = kz_json:set_value(<<"client_token">>, Token, kz_json:new()),
+            crossbar_util:response(JObj, Context)
+    catch
+        'throw':{'api_error', Reason} ->
+            crossbar_util:response('error', <<"braintree api error">>, 400, Reason, Context);
```

Comment:
i will look into specific errors related to it and will update it accordingly :)